### DISCUSSION TITLE
YARN VS NPM?

### DISCUSSION THREAD

**Discussion by AdamMomen** [team-hack/react-components#51] [Q&A]
#### Installation Speed
For comparison, deleting the project `node_modules` folder, and installing over again using both npm and Yarn without a lock file and with a clean cache. This installs 2900 packages in total.

With a constant internet connection speed, the two took different time to install
- npm: 966 seconds
<img src="https://user-images.githubusercontent.com/55929982/101764006-d7593d80-3adf-11eb-93f4-229cd5eae51f.png">

- Yarn:  800 seconds

A detailed comparison between the two [here](https://www.sitepoint.com/yarn-vs-npm/)

**Comment by vijayxtreme** [comment]
I vouch for yarn -- it is a lot better at checking out packages and avoiding package conflicts between versions.
